core/src/executor/evaluate/function.rs

```diff
@@ -596,3 +596,24 @@ pub fn cast<'a>(expr: Evaluated<'a>, data_type: &DataType) -> Result<Evaluated<'
 pub fn extract<'a>(field: &DateTimeField, expr: Evaluated<'_>) -> Result<Evaluated<'a>> {
     Ok(Evaluated::from(Value::try_from(expr)?.extract(field)?))
 }
+
+pub fn point<'a>(x: Evaluated<'_>, y: Evaluated<'_>) -> Result<Evaluated<'a>> {
+    let x = eval_to_float!("point".to_owned(), x);
+    let y = eval_to_float!("point".to_owned(), y);
+
+    Ok(Evaluated::from(Value::Point(crate::data::Point::new(x, y))))
+}
+
+pub fn getx<'a>(name: String, expr: Evaluated<'_>) -> Result<Evaluated<'a>> {
```

Comment:
it would be just good to use `get_x` because Function enum variant name is `GetX`